lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -306,6 +306,7 @@ dictionary Config {
     u64 payment_timeout_sec;
     u32 zero_conf_min_fee_rate_msat;
     u64? zero_conf_max_amount_sat;
+    string? breez_api_key;
```

Comment:
Is this really optional? Seems to me broadcasting certain txs won't work if the user won't explicitly set an API key.

Maybe `connect` should fail if no `Config::breez_api_key` is set?

Reply:
Btw, will make a commit to #489 as that's where I'm tracking those changes